drivers/ethernet/eth_enc424j600.c

```diff
@@ -484,7 +484,6 @@ static void enc424j600_rx_thread(struct enc424j600_runtime *context)
 			}
 		} else {
 			LOG_ERR("Unknown Interrupt, EIR: 0x%04x", eir);
-			continue;
```

Comment:
How this fixes a bug?

Reply:
This explanation makes sense to me, but it should be in the commit message. (Description of how it behaved incorrectly before and how it'll behave correctly after this change.)